it/src/it/env-metadata/expected/kubernetes.yml

```diff
@@ -87,12 +87,12 @@ items:
           ports:
           - containerPort: 8080
             name: http
-            protocol: TCP
+            protocol: tcp
```

Comment:
Not sure where this change comes from, but the protocol has to be UPPERCASE according to the k8s spec at https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#containerport-v1-core

So instead of changing the test we should find out why it is lower case now.

Reply:
Sure, will check by the end of this day ;-)

Reply:
@rhuss : Extremely sorry. I missed a toUpperCase() call while refactoring ;-)